shared/locales/pl/options.json

```diff
@@ -87,5 +87,29 @@
   "invalidURL" : {
     "title" : "Nieprawidłowy adres URL",
     "note" : "The URL the user entered is not a valid website address"
+  },
+  "fireButtonHeading" : {
+    "title" : "Przycisk zabezpieczenia",
```

Comment:
😬 In many places they called Fire Button something else (e.g. "Protection Button" here), while they are referring to it in other descriptions as "Fire Button" (untranslated, see the next section). Not sure if that's OK?

Reply:
FYI, I've rejected the strings where Fire Button was translated asking if it was intended for some reason. We'll check back tomorrow.